src/features/emails/l10n/messageIds.ts

```diff
@@ -8,13 +8,31 @@ export default makeMessages('feat.emails', {
     title: m('Blocked'),
     unsubscribed: m('Unsubscribed'),
   },
+  deliveryProblems: {
+    CONTENT_ERROR: m(
+      'There are problems in the content of your email. Go to the editor in the Compose tab and correct them.'
+    ),
+    EMPTY: m('Your email is empty. Go to the Compose tab to add some content.'),
+    NOT_TARGETED: m(
+      'Your email has no targets. Go to the Targets section in the Overview tab to create a Smart Search that defines your targets.'
+    ),
+    NO_SUBJECT: m(
+      'Your email has no subject line. Go to Settings in the Compose tab to add one.'
+    ),
+    TARGETS_NOT_LOCKED: m(
+      'The targets are not locked. Go to the Ready section un the Overview tab to do this.'
```

Comment:
I spotted this in the screenshot. 😊 

```suggestion
      'The targets are not locked. Go to the Ready section in the Overview tab to do this.'
```

Reply:
![Flag_of_the_United_Nations svg](https://github.com/zetkin/app.zetkin.org/assets/58265097/20674984-154a-445b-b7d0-f64eb27efd11)